src/cljs/orcpub/dnd/e5/views.cljs

```diff
@@ -2904,6 +2904,35 @@
                            weight)]]]]))
               equipment-cfgs))]]]]))))
 
+(defn treasure-section []
+  (let [expanded-details (r/atom {})]
+    (fn [id]
+      (let [mobile? @(subscribe [:mobile?])
+            treasure-cfgs @(subscribe [::char/treasure id])]
+        [:div
+         [:div.flex.align-items-c
+          (svg-icon "cash" 32)
+          [:span.m-l-5.f-w-b.f-s-18 "Treasure"]]
+         [:div
+          [:table.w-100-p.t-a-l.striped
+           [:tbody
+            [:tr.f-w-b
+             {:class-name (if mobile? "f-s-12")}
+             [:th.p-10 "Name"]
+             [:th.p-10 "Qty."]
+             [:th]]
+            (doall
+             (map
+              (fn [[treasure-kw treasure-cfg]]
+                (let [treasure-name (::char-equip/name treasure-cfg)
+                      {:keys [::equip/name] :as treasure} (equip/treasure-map treasure-kw)]
+                  ^{:key treasure-kw}
+                  [:tr.pointer
```

Comment:
Let's take out the mouse cursor change to pointer, since there is nothing to click on....
Change `[:tr.pointer` to just `[:tr`

Reply:
Also take the pointer off the `(defn other-equipment-section-2 []` above.